parlai/crowdsourcing/tasks/model_chat/model_chat_blueprint.py

```diff
@@ -194,7 +194,9 @@ def assert_task_args(
             f'"~" can\'t currently be parsed in the chat data folder path '
             f'{args.blueprint.chat_data_folder}'
         )
-        # TODO: allow ~ to be parsed correctly
+        # Currently Hydra overrides the tilde key at lower levels as described here: https://hydra.cc/docs/next/advanced/override_grammar/basic/#grammar
+        # Thus the TILDE key cannot be used in replacement for $HOME variable
+        # Some hacky solution can probably be achieved but won't be good code so for now this assert is written as a placeholder
```

Comment:
Thanks for adding this comment! This line is a bit long - if you get a Lint error for this line, you might want to split this into 2 lines (`./autoformat.sh` might also fix this for you, not sure)